### PR TITLE
Style

### PR DESCRIPTION
**Pull request by Fatmasiam**
Adding style to the Input component and,
LabeledInput component.

### CODE REVIEW

File: Client/src/components/common/Input/index.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.input{
```
Comment (AsalaKM) [inline Client/src/components/common/Input/index.css]: don't name your class as the general element name !

File: Client/src/components/common/LabeledInput/index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.LabeledInput{
```
Comment (AsalaKM) [inline Client/src/components/common/LabeledInput/index.css]: `.labeledInput` is better

Take a look at this [https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case](url)

**Review by AsalaKM — CHANGES_REQUESTED** [review]

**Review by AsalaKM — APPROVED** [review]

**Comment by AsalaKM** [comment]
Please!
Ask a review from each team members before merge it

**Comment by Fatmasiam** [comment]
Ok

**Review by naremanhilles — APPROVED** [review]

File: Client/src/components/common/Input/index.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
+import './index.css';
 
 const Input = props => {
-  const {inputClassName} = props
   return (
-    <input {...props} className={inputClassName}/>
+    <input {...props} className={'general-input ' + props.className}/>
   )
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/Input/index.js:8]: rewrite  the destruction

**Review by AbdallahAmmar96 — APPROVED** [review]

**Pull request merged**